src/test/suite/providers/formatting/formattingProvider.utils.test.ts

```diff
@@ -315,13 +315,17 @@ suite('FormattingProvider utils', () => {
                 rs: 'str = ""',
             },
             {
-                in: 'str = "; comment with double quote"',
```

Comment:
Let's keep this `in` and update the corresponding `rs`. We can add the new `'str = " ; comment"'` as well for completeness.

Same for line 322 (4 lines below this one)

Reply:
Done.